src/bundler/bundle_v2.zig

```diff
@@ -9581,13 +9577,19 @@ pub const LinkerContext = struct {
         if (chunk.content == .javascript) {
             const sources = c.parse_graph.input_files.items(.source);
             for (chunk.content.javascript.parts_in_chunk_in_order) |part_range| {
-                const source: Logger.Source = sources[part_range.source_index.get()];
+                const source: *Logger.Source = &sources[part_range.source_index.get()];
 
                 const file_path = brk: {
                     if (source.path.isFile()) {
                         // Use the pretty path as the file name since it should be platform-
                         // independent (relative paths and the "/" path separator)
+                        const path = source.path;
```

Comment:
Is copying this intentional?

Reply:
source is a ptr so avoiding accessing it multiple times, but I can just remove this if copying is not cheaper than the accesses.